hooks/install-hooks.sh

```diff
@@ -3,14 +3,20 @@
 HOOK_NAMES="pre-commit"
 HOOK_DIR=$(git rev-parse --show-toplevel)/.git/hooks
 INSTALL_DIR=$(git rev-parse --show-toplevel)/hooks
+COLOR_GREEN=`tput setaf 2`
+COLOR_RESET=`tput sgr0`
 
 for hook in $HOOK_NAMES; do
+    echo -n "Installing $hook hook..."
     # If the hook already exists, is executable, and is not a symlink
     if [ ! -h $HOOK_DIR/$hook -a -x $HOOK_DIR/$hook ]; then
+        echo -n " Hook already exists..."
         mv $HOOK_DIR/$hook $HOOK_DIR/$hook.local
     fi
     # create the symlink, overwriting the file if it exists
     # probably the only way this would happen is if you're using an old version of git
     # -- back when the sample hooks were not executable, instead of being named ____.sample
+    echo -n " Creating symlink."
     ln -s -f $INSTALL_DIR/$hook $HOOK_DIR
+    echo "${COLOR_GREEN} ✓${COLOR_RESET}\n"
```

Comment:
This is literally printing `\n`. Check out this thread: http://stackoverflow.com/questions/8467424/echo-new-line-in-bash-prints-literal-n

(seems like best idea is to go with `printf`)